### DISCUSSION TITLE
Radxa Rock-5C and Radxa 10FHD display

### DISCUSSION THREAD

**Discussion by pterodyne** [Joshua-Riek/ubuntu-rockchip#1083] [Q&A]
Trying to get the Radxa 10FHD display working for the 5c.  I see that there are overlays for the 5a, b, and itx boards.  Ive tried those, and they cause the system to error out during boot.  Using the Radxa builds an rsetup I can make the display work so I know it's not a hardware issue.  I even tried to make a copy of the overlay, renaming it to have a c in the name and linking that in u-boot.  It boots cleanly with that one but the display is not detected or working.

Any thoughts? I downloaded the latest 24.04 desktop image as of today for this test.

Thanks for any thoughts.

Bryan

**Comment by infinite-dark** [comment]
Did you follow the proper procedure for enabling the overlays for Joshua's Ubuntu?

The 5A and 5C are almost exactly the same boards and I've had no problems using overlays for the 5A on the 5C - I tried using the Radxa 4K camera and the Radxa 8HD display and both worked fine.

**Reply by kalyanikasam** [reply]
Hello there,
can you elaborate on detailed procedure to enable overlay. did you change the file name to rock5c or used the same without file name change, i have tried with rasberry pi 5inch also by enabling allnet 5inch display no luck, even tried with radxa 8inch touch screen no luck.
i have followed exact procedure mentioned in joushua reik wiki. please help
